aleph/index/util.py

```diff
@@ -291,27 +291,24 @@ def configure_index(index, mapping, settings):
         }
         config = es.indices.get(index=index).get(index, {})
         settings.get("index").pop("number_of_shards")
-        log.info(f"[{index}] Current settings: {config.get("settings")}")
+        log.info(f"[{index}] Current settings.", settings=config.get("settings"))
```

Comment:
Might as well put `index=index` as an argument to the logger here since we're logging structured data.